vignettes/GGIR.Rmd

```diff
@@ -635,661 +956,1199 @@ knitr::include_graphics("reportexample.jpg")
+    role of sociodemographic factors. Am J Epidemiol. 2014 Mar
+    15;179(6):781-90. doi: 10.1093/aje/kwt330. Epub 2014 Feb 4. PMID:
+    24500862 [link](https://pubmed.ncbi.nlm.nih.gov/24500862/)
 
 If you used the **auto-calibration functionality** then also cite:
 
-4. van Hees VT, Fang Z, et al. Auto-calibration of accelerometer data for free-living physical activity assessment using local gravity and temperature: an evaluation on four continents. J Appl Physiol 2014. [link](https://journals.physiology.org/doi/full/10.1152/japplphysiol.00421.2014)
+4.  van Hees VT, Fang Z, et al. Auto-calibration of accelerometer data
+    for free-living physical activity assessment using local gravity and
+    temperature: an evaluation on four continents. J Appl Physiol 2014.
+    [link](https://journals.physiology.org/doi/full/10.1152/japplphysiol.00421.2014)
 
 If you used the **sleep detection** then also cite:
 
-5. van Hees VT, Sabia S, et al. A novel, open access method to assess sleep duration using a wrist-worn accelerometer, PLoS ONE, 2015 [link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0142533)
+5.  van Hees VT, Sabia S, et al. A novel, open access method to assess
+    sleep duration using a wrist-worn accelerometer, PLoS ONE, 2015
+    [link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0142533)
 
-If you used the **sleep detection without relying on sleep diary** then also cite:
+If you used the **sleep detection without relying on sleep diary** then
+also cite:
 
-6. van Hees VT, Sabia S, et al. Estimating sleep parameters using an accelerometer without sleep diary. Scientific Reports 2018. doi: 10.1038/s41598-018-31266-z. [link](https://www.nature.com/articles/s41598-018-31266-z)
+6.  van Hees VT, Sabia S, et al. Estimating sleep parameters using an
+    accelerometer without sleep diary. Scientific Reports 2018. doi:
+    10.1038/s41598-018-31266-z.
+    [link](https://www.nature.com/articles/s41598-018-31266-z)
 
 If you used the **sleep regularity index** then also cite:
 
-7. Andrew J. K. Phillips, William M. Clerx, et al. Irregular sleep/wake patterns are associated with poorer academic performance and delayed circadian and sleep/wake timing. Scientific Reports. 2017 June 12 [link](https://www.nature.com/articles/s41598-017-03171-4).
+7.  Andrew J. K. Phillips, William M. Clerx, et al. Irregular sleep/wake
+    patterns are associated with poorer academic performance and delayed
+    circadian and sleep/wake timing. Scientific Reports. 2017 June 12
+    [link](https://www.nature.com/articles/s41598-017-03171-4).
 
 ## Copyright for GGIR logo
-The copyright of the GGIR logo lies with Accelting (Almere, The Netherlands), please contact v.vanhees@acceleting.com to ask for permission to use this logo.
 
+The copyright of the GGIR logo lies with Accelting (Almere, The
+Netherlands), please contact
+[v.vanhees\@acceleting.com](mailto:v.vanhees@acceleting.com){.email} to
```

Comment:
Check that `\` in my e-mail address is correct. Does the syntax require this?

Reply:
Yes, it's needed. It looks good in the html version and the link works fine.